webplugin/template/chat.html

```diff
@@ -48,8 +49,9 @@
                 if (skipRouting){
                     defaultSettings["skipRouting"] = !!skipRouting;
                 }
-                
+
                 var kommunicateSettings = {
+                    "textAreaNoFocus" : isInputFocussed == "false",
```

Comment:
name is confusing, i think we should keep just one name and use if everywhere instead of 2 names inputFocus and textAreaNoFocus

Reply:
Made the change and tested it